ints/int21.c

```diff
@@ -555,6 +563,7 @@ void int21()
             {
                 set_flag_C(1);
                 r_ax = errno;   // FIXME
+                uc_reg_write(uc, UC_X86_REG_AX, &r_ax);
```

Comment:
Missing upstream (still bad because of errno, but at least slightly more consistent)